### PR TITLE
Feat implement reach router

### LINKED ISSUE #8: feat: implement reach router for menu

### PR DESCRIPTION
**Pull request by biggicode**

### CODE REVIEW

**Review by MariaTeo — APPROVED** [review]
Nice work! Keep it up 👏🏼🔝

File: src/component/home/home.style.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components'
 import BackgroundPoster from '../../asset/images/poster.jpg'
 
 export const HomeContainer = styled.div`
-  margin-top: 10vh;
+  /* margin-top: 10vh; */
```
Comment (ValentinaVanu) [inline src/component/home/home.style.js:5]: Remove unused code

File: src/component/app/app.component.jsx [hunk 2]
```diff
@@ -2,21 +2,28 @@ import React from 'react'
 import { Provider } from 'react-redux'
 import { ThemeProvider } from 'styled-components'
 import theme from '../../config/theme'
+import * as S from './app.style'
```
Comment (ValentinaVanu) [inline src/component/app/app.component.jsx]: Why import * (all) as S if there is only one export ?

**Review by ValentinaVanu — CHANGES_REQUESTED** [review]

File: src/component/app/app.component.jsx (hunk 2 above)
Comment (biggicode) [reply src/component/app/app.component.jsx]: cause will be more

**Review by marianzburlea — APPROVED** [review]
💯

**Review by ValentinaVanu — COMMENTED** [review]
👍

**Review by ValentinaVanu — APPROVED** [review]

**Pull request merged**